lib/cog/repository/rules.ex

```diff
@@ -176,16 +176,7 @@ defmodule Cog.Repository.Rules do
     case find_or_create_rule(ast, command) do
       {:ok, rule} ->
         grant_permissions(rule, permissions)
-
-        # If we have disabled a bundle rule, but then add the same
-        # rule ourselves as a site rule, we need to ensure it's
-        # enabled here.
-        rule = if Cog.Repository.Bundles.is_site_version?(bundle_version) do
-          set_rule_status(rule, :enabled)
-        else
-          rule
-        end
-
+        set_rule_status(rule, :enabled)
```

Comment:
No, we want to only do this if we're doing a site rule. Otherwise if you install a new version of a bundle that has a rule you've previously removed, this will resurrect it.